tests/tests/test_bundle.py

```diff
@@ -125,6 +125,41 @@ def side_effect(*args, **kwargs):
         self.assertEqual(process_mock.communicate.call_count, 1)
 
     @mock.patch('subprocess.Popen')
+    def test_bundle_skip_source_maps_suprocess(self, mock_subproc_popen):
+        """
+        Test that bundling calls the correct subprocess command
+        """
+
+        app_name = 'app/dummy'
+
+        def side_effect(*args, **kwargs):
+            _bundle(app_name)
+            return ('output', 'error')
+
+        # mock Popen/communicate
+        process_mock = mock_Popen(mock_subproc_popen, side_effect=side_effect)
+
+        # Bundle app/dummy
+        system = System(skip_source_maps=True)
+        system.bundle('app/dummy')
+        self.assertEqual(mock_subproc_popen.call_count, 1)
+        command = mock_subproc_popen.call_args[0][0]
+        outfile = os.path.join(settings.STATIC_ROOT, 'SYSTEMJS/app/dummy.js')
+        self.assertEqual(command,
+                         'jspm bundle app/dummy {0} --skip-source-maps'.format(outfile))
+
+        mapfile = os.path.join(settings.STATIC_ROOT,
+                               'SYSTEMJS/{0}.js.map'.format(app_name))
+        self.assertFalse(os.path.isfile(mapfile))
```

Comment:
the idea is of course nice, but this assertion has little value since the actual bundling is mocked away. Asserting that the option is indeed passed on the suprocess command is enough - it's `jspm`/`SystemJS`' duty to ensure that the source map is indeed skipped.

I would remove this assertion to avoid confusion, and maybe add my above comment as a comment in the test.